docker/addons/certs/docker-compose.yml

```diff
@@ -14,66 +14,66 @@ volumes:
     container_name: magistrala-certs
     depends_on:
       - am-certs
     restart: on-failure
     networks:
       - magistrala-base-net
     ports:
-      - ${MG_CERTS_HTTP_PORT}:${MG_CERTS_HTTP_PORT}
+      - ${SMQ_CERTS_HTTP_PORT}:${SMQ_CERTS_HTTP_PORT}
     environment:
-      MG_CERTS_LOG_LEVEL: ${MG_CERTS_LOG_LEVEL}
-      MG_CERTS_SIGN_CA_PATH: ${MG_CERTS_SIGN_CA_PATH}
-      MG_CERTS_SIGN_CA_KEY_PATH: ${MG_CERTS_SIGN_CA_KEY_PATH}
-      MG_CERTS_VAULT_HOST: ${MG_CERTS_VAULT_HOST}
-      MG_CERTS_VAULT_NAMESPACE: ${MG_CERTS_VAULT_NAMESPACE}
-      MG_CERTS_VAULT_APPROLE_ROLEID: ${MG_CERTS_VAULT_APPROLE_ROLEID}
-      MG_CERTS_VAULT_APPROLE_SECRET: ${MG_CERTS_VAULT_APPROLE_SECRET}
-      MG_CERTS_VAULT_THINGS_CERTS_PKI_PATH: ${MG_CERTS_VAULT_THINGS_CERTS_PKI_PATH}
-      MG_CERTS_VAULT_THINGS_CERTS_PKI_ROLE_NAME: ${MG_CERTS_VAULT_THINGS_CERTS_PKI_ROLE_NAME}
-      MG_CERTS_HTTP_HOST: ${MG_CERTS_HTTP_HOST}
-      MG_CERTS_HTTP_PORT: ${MG_CERTS_HTTP_PORT}
-      MG_CERTS_HTTP_SERVER_CERT: ${MG_CERTS_HTTP_SERVER_CERT}
-      MG_CERTS_HTTP_SERVER_KEY: ${MG_CERTS_HTTP_SERVER_KEY}
-      MG_CERTS_DB_HOST: ${MG_CERTS_DB_HOST}
-      MG_CERTS_DB_PORT: ${MG_CERTS_DB_PORT}
-      MG_CERTS_DB_PASS: ${MG_CERTS_DB_PASS}
-      MG_CERTS_DB_USER: ${MG_CERTS_DB_USER}
-      MG_CERTS_DB_NAME: ${MG_CERTS_DB_NAME}
-      MG_CERTS_DB_SSL_MODE: ${MG_CERTS_DB_SSL_MODE}
-      MG_CERTS_DB_SSL_CERT: ${MG_CERTS_DB_SSL_CERT}
-      MG_CERTS_DB_SSL_KEY: ${MG_CERTS_DB_SSL_KEY}
-      MG_CERTS_DB_SSL_ROOT_CERT: ${MG_CERTS_DB_SSL_ROOT_CERT}
-      MG_CERTS_SDK_HOST: ${MG_CERTS_SDK_HOST}
-      MG_CERTS_SDK_CERTS_URL: ${MG_CERTS_SDK_CERTS_URL}
-      MG_CERTS_SDK_TLS_VERIFICATION: ${MG_CERTS_SDK_TLS_VERIFICATION}
+      SMQ_CERTS_LOG_LEVEL: ${SMQ_CERTS_LOG_LEVEL}
+      SMQ_CERTS_SIGN_CA_PATH: ${SMQ_CERTS_SIGN_CA_PATH}
+      SMQ_CERTS_SIGN_CA_KEY_PATH: ${SMQ_CERTS_SIGN_CA_KEY_PATH}
+      SMQ_CERTS_VAULT_HOST: ${SMQ_CERTS_VAULT_HOST}
+      SMQ_CERTS_VAULT_NAMESPACE: ${SMQ_CERTS_VAULT_NAMESPACE}
+      SMQ_CERTS_VAULT_APPROLE_ROLEID: ${SMQ_CERTS_VAULT_APPROLE_ROLEID}
+      SMQ_CERTS_VAULT_APPROLE_SECRET: ${SMQ_CERTS_VAULT_APPROLE_SECRET}
+      SMQ_CERTS_VAULT_CLIENTS_CERTS_PKI_PATH: ${SMQ_CERTS_VAULT_CLIENTS_CERTS_PKI_PATH}
+      SMQ_CERTS_VAULT_CLIENTS_CERTS_PKI_ROLE_NAME: ${SMQ_CERTS_VAULT_CLIENTS_CERTS_PKI_ROLE_NAME}
+      SMQ_CERTS_HTTP_HOST: ${SMQ_CERTS_HTTP_HOST}
+      SMQ_CERTS_HTTP_PORT: ${SMQ_CERTS_HTTP_PORT}
+      SMQ_CERTS_HTTP_SERVER_CERT: ${SMQ_CERTS_HTTP_SERVER_CERT}
+      SMQ_CERTS_HTTP_SERVER_KEY: ${SMQ_CERTS_HTTP_SERVER_KEY}
+      SMQ_CERTS_DB_HOST: ${SMQ_CERTS_DB_HOST}
+      SMQ_CERTS_DB_PORT: ${SMQ_CERTS_DB_PORT}
+      SMQ_CERTS_DB_PASS: ${SMQ_CERTS_DB_PASS}
+      SMQ_CERTS_DB_USER: ${SMQ_CERTS_DB_USER}
+      SMQ_CERTS_DB_NAME: ${SMQ_CERTS_DB_NAME}
+      SMQ_CERTS_DB_SSL_MODE: ${SMQ_CERTS_DB_SSL_MODE}
+      SMQ_CERTS_DB_SSL_CERT: ${SMQ_CERTS_DB_SSL_CERT}
+      SMQ_CERTS_DB_SSL_KEY: ${SMQ_CERTS_DB_SSL_KEY}
```

Comment:
Why don't we reuse the smq addons ?

Reply:
Solved by #76

Reply:
Still we need `addons/certs/docker-compse.yml` ?